tests/integration-test/src/tests/basic.rs

```diff
@@ -7,6 +7,11 @@ use rand::Rng;
 use super::{integration_test, IntegrationTest};
 use crate::tests::utils::*;
 
+#[integration_test]
+fn test_bpfctl_helptext() {
+    let _ = get_bpfctl_list_helptext();
```

Comment:
No need to even use let here if we don't care about the string

Reply:
Just unwrap the result

Reply:
Reworked based on Andre's comment.